File: v1/src/main/java/com/api/v1/borrower/helpers/BorrowerResponseMapper.java
package com.api.v1.borrower.helpers;

import com.api.v1.borrower.domain.Borrower;

class BorrowerResponseMapper {

    public static BorrowerResponse mapFromBorrower(Borrower borrower) {
        return new BorrowerResponse(
            borrower.getFullName(), 
            borrower.getSsn(), 
            borrower.getEmail(),
            borrower.getAddress(), 
            borrower.getPhoneNumber(), 
            borrower.getGender(),
            borrower.getCreatedAt(),
            borrower.getUpdatedAt()
        );
    }
    
}

File: koltlin/v1/src/main/kotlin/com/api/v1/borrower/helpers/BorrowerResponseMapper.kt
package com.api.v1.borrower.helpers

import com.api.v1.borrower.domain.Borrower

class BorrowerResponseMapper {

    companion object {
        fun map(borrower: Borrower): BorrowerResponseDto {
            return BorrowerResponseDto(
                borrower.getFullName(),
                borrower.ssn,
                borrower.birthDate,
                borrower.email,
                borrower.address,
                borrower.gender,
                borrower.phoneNumber,
                borrower.createdAt,
                borrower.updatedAt
            )
        }
    }

}